matchms/importing/__init__.py

```diff
@@ -30,6 +31,8 @@
     "load_from_mzml",
     "load_from_mzxml",
     "load_from_usi",
+    "load_spectra",
```

Comment:
I was afraid this might cause some circular import issues, but if the tests pass that is a good addition indeed!